Dockerfile

```diff
@@ -1,5 +1,5 @@
 # First, get the elixir dependencies within an elixir container
-FROM hexpm/elixir:1.11.4-erlang-23.3.4.18-alpine-3.16.2 AS elixir-builder
+FROM hexpm/elixir:1.17.3-erlang-27.1-debian-bullseye-20240926-slim as elixir-builder
```

Comment:
hmm I assumed because CI ran it worked, but good point I did not notice I had switch from Alpine I'll change it back. I also didn't think I was allowed docker for mac on the machine because of licensing, did I hallucinate that?

Reply:
Docker Desktop is not allowed, correct. Colima is the replacement for running containers locally on Macs: https://github.com/mbta/technology-docs/blob/main/rfcs/accepted/0010-docker-desktop-replacement.md

Reply:
Great I added readme instructions too 👍